xlml/utils/metric.py

```diff
@@ -522,5 +530,8 @@ def process_metrics(
 
   print("Test run rows:", test_run_rows)
 
-  if is_valid_entry():
-    bigquery_metric.insert(test_run_rows)
+  # if is_valid_entry():
```

Comment:
Did you mean to remove or uncomment this line?

Reply:
Thanks Will for taking a look! I am still in progress on this PR, and I think it has automatically sent to your inbox due to code owner file :)

Reply:
Thanks! Now it's ready for review.